src/main/webapp/publico/candidacy/mobility/viewCandidacy.jsp

```diff
@@ -171,6 +171,10 @@
 	<p class="mbottom05"><em><bean:message key="message.ist.conditions.note" arg0="<%=org.fenixedu.academic.domain.organizationalStructure.Unit.getInstitutionAcronym()%>" bundle="CANDIDATE_RESOURCES"/></em></p>
 </logic:equal>
 
+<logic:notEqual name="individualCandidacyProcess" property="allRequiredFilesUploaded" value="false">
```

Comment:
avoid double negative logic :)
https://derickbailey.com/2014/03/27/double-negatives-in-code-dont-not-avoid-double-negative-boolean-logic/